### PR TITLE
feature/webstore-023

### PR DESCRIPTION
**Pull request by alexMIRON1**

### CODE REVIEW

File: paymenthandler/README.md [hunk 1]
```diff
@@ -46,3 +46,33 @@ create schema payment;
 2. Click button `Test connection`, if you all do correctly, connection will be set up 
 3. Now, you could execute the same queries to create database and schema
 ```
+## Business logic
+
```
Comment (sokimaaa) [inline paymenthandler/README.md:50]: do we really need to implement bank system? let's just mock it and store customer card, and its balance,
so we could rewrite as
(1) Customer adds a new card for payments
(2) Customer wants to remove card from our service (he doesn't wanna use it anymore)

Comment (alexMIRON1) [reply paymenthandler/README.md:50]: But how we will know that card is valid, that customer has enough balance on card. And how we will know that our transaction was successfull completed. Do you wanna mock the result? beacuse if yes , so all transaction will be successfull.

Comment (sokimaaa) [reply paymenthandler/README.md:50]: we're gonna count that all cards have unlimited balance and valid, but for buying on our store, people have to use our own currency, e.g. Credits.

To get our currency, make some 'Convertation button'. I think you know about games that have built-in currency and how it works.

So, now all transaction will be with our currency and we could write logic you want.

Comment (sokimaaa) [reply paymenthandler/README.md:50]: P.S. Anyway we will top-up balance by button click and card will have unlimited balance.

About validation, You are still able to add it, e.g. user entered characters, but expected only digits.

Comment (sokimaaa) [reply paymenthandler/README.md:50]: I'd say that we could refuse idea with real cards at all, and use our own currency.

Comment (alexMIRON1) [reply paymenthandler/README.md:50]: Ok, I get it. We just replace card, with our own balance - currency

**Review by sokimaaa — APPROVED** [review]

**Pull request merged**